lib/killbill/helpers/active_merchant/payment_plugin.rb

```diff
@@ -532,7 +532,8 @@ def build_am_credit_card(cc_number, attributes, pm=nil)
           tokenization_attributes = {
               :eci => Utils.normalized(attributes, :eci),
               :payment_cryptogram => Utils.normalized(attributes, :payment_cryptogram),
-              :transaction_id => Utils.normalized(attributes, :transaction_id)
+              :transaction_id => Utils.normalized(attributes, :transaction_id),
+              :source => Utils.normalized(attributes, :source)
```

Comment:
we tried to use ```source``` in comments on this PR ? https://github.com/killbill/killbill-cybersource-plugin/pull/8